### PR TITLE
[fix](nereids)add physical property in post processor

### PR DESCRIPTION
**Pull request by englefly**
## Proposed changes
Post rule ProjectAggregateExpressionsForCse may add new Project node into plan, but does not set physical properties for the new project, and hence breaks the function of local shuffle.
Issue Number: close #xxx

### CODE REVIEW

**Comment by englefly** [comment]
run buildall

**Comment by morrySnow** [comment]
add desc please

File: fe/fe-core/src/main/java/org/apache/doris/nereids/trees/plans/physical/PhysicalProject.java [hunk 1]
```diff
@@ -63,6 +63,11 @@ public PhysicalProject(List<NamedExpression> projects, LogicalProperties logical
         this(projects, Optional.empty(), logicalProperties, child);
     }
 
+    public PhysicalProject(List<NamedExpression> projects, LogicalProperties logicalProperties,
+            PhysicalProperties physicalProperties, CHILD_TYPE child) {
+        this(projects, Optional.empty(), logicalProperties, child);
```
Comment (morrySnow) [inline fe/fe-core/src/main/java/org/apache/doris/nereids/trees/plans/physical/PhysicalProject.java:68]: input parameter physicalProperties is not used

**Review by morrySnow — COMMENTED** [review]
add test case please

**Comment by englefly** [comment]
run buildall

**Review by morrySnow — APPROVED** [review]

**Review by starocean999 — APPROVED** [review]

**Pull request merged**